components/simple-v-stepper/SimpleStepper.vue

```diff
@@ -1,48 +1,47 @@
 <template>
-    <div class="simple-stepper">
-        <div class="stepper-progress-bar">
+    <section class="simple-stepper">
+        <header class="progress-bar">
             <div v-for="(step, index) in stepsDescription"
                  v-bind:class="stepClass(index)">
-                <div class="stepper-step-wrapper" @click.prevent="moveNextStep(index + 1)">
-                    <div class="step-circle">
+                <button class="step-navigation-button" @click.prevent="moveNextStep(index + 1)">
+                    <div class="step-navigation-icon">
                         <span> {{index +1}} </span>
                     </div>
-                    <div class="step-title">
+                    <span class="step-navigation-title">
                         {{step.name}}
-                    </div>
-                </div>
-                <div v-if="index > 0" class="bar-left"></div>
-                <div v-if="index + 1  < stepsDescription.length" class="bar-right"></div>
+                    </span>
+                </button>
+                <!-- <div v-if="index > 0" class="bar-left"></div> -->
+                <div v-if="index + 1  < stepsDescription.length" class="step-navigation-deco"></div>
             </div>
-        </div>
+        </header>
 
-        <div class="step">
+        <section class="step-wrapper">
             <slot v-for="index in stepsDescription.length"
```

Comment:
Tutaj tak naprawdę wyświetlamy tylko jeden `slot`, więc można by pokusić o wywalenie `v-for`a i `v-if`a i zrobić po prostu `:name = 'step-' + stepNumber` (chyba, że nie działają tak sloty)